internal/expr/query.go

```diff
@@ -12,12 +12,21 @@ func (qe *QueryExpr) QueryArgs() []any {
 	return qe.args
 }
 
+func (qe *QueryExpr) SQL() string {
+	return qe.pe.sql(qe.sc)
+}
+
+func (qe *QueryExpr) IsTemp() bool {
+	return len(qe.sc.sliceLens) > 0
+}
```

Comment:
Can we add a comment here saying that this function basically checks if the statement can be re-prepared?